app/views/notification_mailer/new_response.text.erb

```diff
@@ -0,0 +1,18 @@
+<%= _('You have a new response to the {{law_used_full}} request',
+      :law_used_full => @info_request.law_used_human(:full).html_safe) %>
+'<%= raw @info_request.title %>' <%=_('that you made to') %>
```

Comment:
Fixed in 88b6dd5 - I kept this separate rather than as a fixup because it's present in the old view I copied this from and this is supposed to be a 'temporary' duplication, so this fix changes both.